### PR TITLE
FHAC-246: Fix to display the CDA document in HTML and few other fixes

### PR DESCRIPTION
**Pull request by nareshcgi**
Fixed the code to display the CDA document in HTML
Fixed the page to use the attachment PDF and Image data from the backend  bean not from browser cache.
Fixed the document metadata and attachment Modal window title

### CODE REVIEW

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/PatientSearchBean.java [hunk 1]
```diff
@@ -653,4 +663,31 @@ public void setDisplayOrganizationName(String displayOrganizationName) {
         this.displayOrganizationName = displayOrganizationName;
     }
 
+    /**
+     * @return the documentTypeName
+     */
+    public String getDocumentTypeName() {
+        //
```
Comment (jsmith2) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/PatientSearchBean.java]: remove empty comment line.

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/PatientSearchBean.java [hunk 2]
```diff
@@ -71,20 +71,14 @@
...
-    private static final String CONTENT_TYPE_TEXT_XML = "text/xml";
-    private static final String CONTENT_TYPE_TEXT_PLAIN = "text/plain";
-    private static final String CONTENT_TYPE_TEXT_HTML = "text/html";
-    private static final String CONTENT_TYPE_APPLICATION_XML = "application/xml";
-    private static final String CONTENT_TYPE_APPLICATION_XHTML_XML = "application/xhtml+xml";
-    private static final String CONTENT_TYPE_APPLICATION_OCTET_STREAM = "application/octet-stream";
-    private static final String CONTENT_TYPE_APPLICATION_PDF = "application/pdf";
+    private static final String PATIENT_SEARCH_LOAD_MESSAGE = "Searching Patient....";
+    private static final String DOCUMENT_SEARCH_LOAD_MESSAGE = "Searching Patient Documents....";
+    private static final String DISPLAYING_LOAD_MESSAGE = "Retrieving Patient Document....";
```
Comment (jsmith2) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/PatientSearchBean.java]: Why were these strings added back in?  They aren't being used.

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/PatientSearchBean.java [hunk 3]
```diff
@@ -653,4 +663,31 @@ public void setDisplayOrganizationName(String displayOrganizationName) {
...
+        }
+        return "Unknown Document";
+    }
+
+    /**
+     * @return the documentInfoModalWindowHeader
+     */
+    public String getDocumentInfoModalWindowHeader() {
+        String localAuthorPerson = getSelectedCurrentDocument().getAuthorPerson() == null ? "Unknown" : getSelectedCurrentDocument().getAuthorPerson();
+        return getDocumentTypeName() + " for " + getSelectedCurrentPatient().getName() + " Created by " + localAuthorPerson + " on " + getCreationTimeUiDisplay();
```
Comment (jsmith2) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/PatientSearchBean.java]: Can we shorten this to just doc title and patient name.  Looked a little strange when I displayed it.

Thinks it should just show: "Document Title, John Doe".

Kerry had expressed the same comment on this to me too.

**Comment by alameluchidambaram** [comment]
Reviewed PR and good with the merge.
